src/custom-elements/src/constants/constants.ts

```diff
@@ -59,4 +59,4 @@ export type MouseCursorPosition = { x: number, y: number };
 export const APP_LAYOUT_FORMATS = ['base', 'no-margin', 'fixed-width'];
 export type AppLayoutFormats = typeof APP_LAYOUT_FORMATS[number];
 
-export const TABS_SEE_MORE_DEFAULT_MESSAGE = 'See more...';
+export const TABS_SEE_MORE_DEFAULT_MESSAGE = 'More...';
```

Comment:
Let's change 'More...' to 'More' for consistency with the portal style guide.

```suggestion
export const TABS_SEE_MORE_DEFAULT_MESSAGE = 'More';
```